src/components/ShareSaveQuery/ui.js

```diff
@@ -11,3 +12,11 @@ export const CustomLightButton = styled(LightButton)`
   min-width: 75px;
   font-size: 10px;
```

Comment:
For deprecated components in the future, let's add deprecation messages, maybe like so:
```
export const CustomLightButton = styled(LightButton)`
  ${() => console.warn("CustomLightButton has been deprecated (src/components/FileRepoSidebar/ui.js)")}
  min-width: 75px;	  min-width: 75px;
  font-size: 10px;	  font-size: 10px;
```

Reply:
of course with a little more engineering so we don't have these in prod :P